adapter/repository/in-memory-simple/src/main/java/org/mobilitydata/gtfsvalidator/db/InMemoryGtfsDataRepository.java

```diff
@@ -74,4 +75,27 @@ public Route addEntity(final Route newRoute) throws SQLIntegrityConstraintViolat
             return newRoute;
         }
     }
+
+    private final Map<String, Trip> tripCollection = new HashMap<>();
+
+    @Override
+    public Map<String, Trip> getTripCollection() {
```

Comment:
Remove from public API as it is used only for test purposes. Rewrite test to use getTripById accessor instead to validate the whole map content